ios/Classes/CleverPushPlugin.m

```diff
@@ -99,6 +101,17 @@ - (void)getNotifications:(FlutterMethodCall *)call withResult:(FlutterResult)res
     result(storedNotifications);
 }
 
+- (void)getRemoteNotifications:(FlutterMethodCall *)call withResult:(FlutterResult)result {
```

Comment:
Could we rename this method to `getNotifications` so we have the current `getNotifications` with no arguments and this `getNotifications` with the boolean and the callback argument?
So we have a similar method signature like in the native SDKs and not a new method.

Reply:
Also, I think we need to update the `getNotifications` method which was there before as well so it converts the CPNotifications to Dicts.

Reply:
It is possible to rename the method name in the `CleverPushPlugin.m` files but it is giving me an error in the `cleverpush_flutter.dart` when we define the same method with different argument. and ultimately we are using the dart file methods in flutter.